tico/utils/convert.py

```diff
@@ -294,10 +131,10 @@ def convert(
         logger.fatal(
             "Your model is in TRAINING MODE. PLEASE CHECK IF YOU FORGOT `model.eval()`."
         )
-
+    
     with torch.no_grad():
         exported_program = export(mod, args, kwargs, strict=strict)
-
+    breakpoint()
```

Comment:
@mhs4670go @glistening Run ./ccex test -m LlamaWithKVCache and see this pattern match's working.